addons/udes_stock/tests/test_picking_batch.py

```diff
@@ -1219,7 +1222,7 @@ def test08_check_computing_simple(self):
             self.batch01.confirm_picking()
 
         mocked_compute.assert_called_with(self.batch01)
-        self.assertEqual(
+        self.assertGreaterEqual(
```

Comment:
Does this mean that the function is called more than once now?

Reply:
yes, it can be called but may exit because of state_lock